src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -200,15 +209,35 @@ export class AKSClusterSettingsComponent
     } else {
       this.control(Controls.Location)
         .valueChanges.pipe(debounceTime(this._debounceTime))
-        .pipe(switchMap((location: string) => this._getAKSVmSizes(location)))
+        .pipe(tap(_ => this._clearVmSize()))
+        .pipe(filter(value => !!value))
+        .pipe(
+          switchMap((location: string) => {
+            this.control(Controls.VmSize).enable();
```

Comment:
Info:
This part is intentional as control should only be disabled
**Point-1:**
- when no value existed (valid/invalid) value -> as soon user enters and API call is kicked off -> `loading` spinner will be shown on enabled state.
**Point-2**
- I didn't intentionally made `this.vmSizeLabel = VMSizeState.Loading;` since on enabled state
than label + `mat-error` shows -> `loading is required`  as label Name is used as label + mat-error too in `km-autocomplete` therefore a sort of `hack` is added to keep label same
When ready or Empty in enabled state